rust/src/lib.rs

```diff
@@ -424,10 +424,64 @@ pub fn hirschberg(
     Ok((aligned_seq_one, aligned_seq_two))
 }
 
+fn score_pair(
+    seq_one_val: i64,
+    seq_two_val: i64,
+    match_score: f64,
+    mismatch_score: f64,
+    indel_score: f64,
+    gap_val: i64,
+) -> f64 {
+    if seq_one_val == seq_two_val {
+        match_score
+    } else if (seq_one_val == gap_val) || (seq_two_val == gap_val) {
+        return indel_score;
```

Comment:
I don't think you need these return statements

Reply:
Oh weird, the linter got rid of one of them but not the rest